doc/classes/Line2D.xml

```diff
@@ -210,44 +217,76 @@
 		<member name="end_cap_mode" type="int" setter="set_end_cap_mode" getter="get_end_cap_mode" enum="Line2D.LineCapMode">
+			Controls the style of the line's last point. Use [code]LINE_CAP_*[/code] constants. Default value: [code]LINE_CAP_NONE[/code].
 		</member>
 		<member name="gradient" type="Gradient" setter="set_gradient" getter="get_gradient">
+			The gradient is drawn through the whole line from start to finish. The default color will not be used if a gradient is set.
 		</member>
 		<member name="joint_mode" type="int" setter="set_joint_mode" getter="get_joint_mode" enum="Line2D.LineJointMode">
+			The style for the points inbetween the start and the end.
 		</member>
 		<member name="points" type="PoolVector2Array" setter="set_points" getter="get_points">
+			The points that form the lines. The line is drawn between every point set in this array.
 		</member>
 		<member name="round_precision" type="int" setter="set_round_precision" getter="get_round_precision">
+<<<<<<< HEAD
+			The smoothness of the rounded joints and caps This is only used if a cap or joint is set as round.
 		</member>
 		<member name="sharp_limit" type="float" setter="set_sharp_limit" getter="get_sharp_limit">
+			The direction difference in radians between vector points. This value is only used if [code]joint mode[/code] is set to [code]LINE_JOINT_SHARP[/code]
 		</member>
 		<member name="texture" type="Texture" setter="set_texture" getter="get_texture">
+			The texture used for The line's texture. Uses [code]texture_mode[/code] for drawing style.
 		</member>
 		<member name="texture_mode" type="int" setter="set_texture_mode" getter="get_texture_mode" enum="Line2D.LineTextureMode">
+			The style to render the [code]texture[/code] on the line. Use [code]LINE_TEXTURE_*[/code] constants. Default value: [code]LINE_TEXTURE_NONE[/code].
+=======
+			Sets how smooth the rounded joints and caps will be. This is only used if a cap or joint is set as round.
+		</member>
+		<member name="sharp_limit" type="float" setter="set_sharp_limit" getter="get_sharp_limit">
+			This sets the direction difference in radians between vector points. This value is only used if [code]joint mode[/code] is set to [code]LINE_JOINT_SHARP[/code]
+		</member>
+		<member name="texture" type="Texture" setter="set_texture" getter="get_texture">
+			The texture used for the line. The way it's rendered on the line is set in [code]texture_mode[/code].
+		</member>
+		<member name="texture_mode" type="int" setter="set_texture_mode" getter="get_texture_mode" enum="Line2D.LineTextureMode">
+			The style to render the [code]texture[/code] on the line.
+>>>>>>> Line2Ddocs
 		</member>
 		<member name="width" type="float" setter="set_width" getter="get_width">
+			The width of the line.
 		</member>
 	</members>
 	<constants>
 		<constant name="LINE_JOINT_SHARP" value="0">
+			The joints of the line will be pointy. If Sharp Limit is greater than the rotation of a joint it becomes a bevel joint instead.
```

Comment:
According to [documentation writing guidelines](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html#use-s-to-show-possession), one should write "line's joints" instead of "joints of the line". 
To apply everywhere.